src/parse-args.js

```diff
@@ -38,8 +39,10 @@ module.exports = function (opts) {
   }
 
   try {
-    if (fs.statSync(opts._[0]).isDirectory()) {
+    if (fs.statSync(opts._[0]).isDirectory() && opts._.length !== 2) {
```

Comment:
We return [early in this file](https://github.com/TimothyStiles/dat/blob/66b6e77300cee49485a19679942c50032c14f081/src/parse-args.js#L23) if the length is two, so this isn't needed. I wanted to avoid two `fs.statSync` on the same path, so I did an if/else. I'm not really sure if there is a case where `stat.isDirectory()` and `stat.isFile()` are both false but I figure we only need to explicitly check for file.